src/websocket.rs

```diff
@@ -51,37 +50,34 @@ impl AdminWebsocket {
             info!("returning agent key from memory");
             return Ok(key);
         }
-
+        let force = match env::var("FORCE_RANDOM_AGENT_KEY") {
+            Ok(f) => !f.is_empty(),
+            Err(_) => true,
```

Comment:
Can we also just add a small test here stating the expectations around when the default is used?